docs/data_management/tutorial/globus_group_tutorial.md

```diff
@@ -0,0 +1,399 @@
+
+    ![GCP Setup window with Log In button.](./images/gg-gcp-install/win/006-authenticate.png)
+
+    Note that if you uninstall and reinstall GCP, you will need to complete this process again. You should not need to repeat this process otherwise.
+
+1. Grant the required consents. This is required to set up your computer as a Collection. Also provide a name for your consents. We recommend choosing a name that is short, memorable, and related to the purpose for the Collection.
+
+    ![GCP Setup form with required consents and Collection name field.](./images/gg-gcp-install/common/consents.png)
+
+1. Enter the details for your GCP Collection, and click save to continue. The following list describes the fields in the form shown below.
+
+    - **Owner Identity:** is the person responsible for this Collection. This field should already be filled with UAB Campus or XIAS email address. If not, please that email address here.
+    - **Collection Name:** is the name for the Collection. This should be filled with the name of the Collection from the previous step.
+    - **Description:** Feel free to enter descriptive information about the Collection here. This information will be displayed in the Globus Web App when the Collection is viewed by others.
+    - **High Assurance:** Only check this box if the Collection has or will have PHI, HIPAA, or other protected data. If this is the case, please ensure that you have already completed a risk assessment with UAB Enterprise IT.
+
+    ![GCP Setup Collection details](./images/gg-gcp-install/common/collection-details.png)
+
+1. GCP Setup is now complete on your computer. Your computer is now serving Globus Collection and may be used to transfer data. Click "Exit Setup" to close the window.
+
+1. After installation, you should see a lowercase letter "g" in a circle in your Windows system tray, typically at the bottom-right of the display. If you do not, try finding the Globus Connect Personal application in your start menu and starting the application.
+
+    ![GCP Icon in Windows system tray.](./images/gg-gcp-install/win/009-system-tray-icon.png)
+
+Continue on with [How Do I Choose Specific Folders Using Globus Connect Personal?](#how-do-i-choose-specific-folders-to-share-using-globus-connect-personal).
+
+[Return to the top](#globus-tutorials-for-research-groups).
+
+### Installing GCP On MacOS
+
+The following steps will guide you to install Globus Connect Personal (GCP) on your computer running MacOS.
+
+1. Navigate to the [Globus Connect Personal official page](https://www.globus.org/globus-connect-personal) and scroll down to find the GCP MacOS version. Click on the "INSTALL NOW" button in the red box in the image below to be taken to the official installation instructions and download link for GCP for MacOS.
```

Comment:
It would be better to highlight the 'INSTALL NOW' button with a red box or remove the text "red box" from the main text. For consistency with the macOS installation, marking the button with a red box would be preferable.